src/Win95/Window.c

```diff
@@ -476,6 +461,17 @@ int Window_Main(HINSTANCE hInstance, HINSTANCE hPrevInstance, PSTR pCmdLine, int
     Window_SetGUID((GUID*)Window_UUID);
 
     glfwInit();
+
+    { // Core compatibility for RenderDocs
+        // 4.3 for glDebugMessageCallback
```

Comment:
why do you use 4.3 and not 4.5? some specific hardware that you want to support? 🙂 if you want to go the extra compatible way, 3.3 seems better because that also works on really old gpu drivers on old laptops. most used functionality is 3.3 anyway

Reply:
I was thinking that we'll need bindless textures available in 4.4, and DSA from 4.5 is a nice quality of life as well. I'll bump the version in glad